content/pages/privacy-policy.md

```diff
@@ -0,0 +1,372 @@
+
+- changes to planned volunteer work programmes that you may be taking part in
+- the positive impact you have on our work
+
+#### e) If you have made a general enquiry, which we think is better handled by a local Trust, we will pass this enquiry on
+
+The lawful basis for processing your data for these purposes may be 'contractual' and 'legitimate interest' (when sending you information about our work).
+
+### 4. What kind of personal data do we collect? How do we collect it?
+
+#### a) Basic information
+
+We will usually collect basic information about you, including your name, postal address, telephone number, email address and your bank details if you are supporting us financially.
+
+Most of the time, we collect this data from you directly. Sometimes this is in person; other times, it is over the telephone, in writing or through an email. Occasionally we obtain information, such as your telephone number or other contact details, from external sources (only where you have given permission for such information to be shared).
+
+#### b) Getting to know you better
+
+We also collect information about you that helps us to get to know you better. This may include:
+
+- records of donations you've made
+- your preferences of how you would like us to contact you
+- ways you've helped us through volunteering your time
+- records of events you've attended, or campaigns or activities that you've been involved in
+
+Sometimes we will collect other information about you such as your date of birth and gender. When we do so, we will be very clear as to why we are collecting such information, and we will only do so with your specific consent and permission.
+
+Once again, most of the time we collect this data from you directly.
+
+We may also collect demographic and consumption data generated through geodemographic tools (such as CACI Acorn), as well as information related to your wealth. This may include information from public registers and other publicly available sources such as Companies House, newspapers and magazines. If you do not wish your data to be collected in any of these ways, or have questions about them, please contact us.
+
+Telephone: 01636 677711 \
+Email: [enquiries@wildlifetrusts.org](mailto:enquiries@wildlifetrusts.org) \
+The Kiln, Mather Road, Newark, NG24 1WT
+
+Our office hours are Monday – Friday, 9am – 5pm.
+
+Other ways in which we collect personal data to get to know you better include:
+
+##### i) Our website
+
+Our website uses 'cookies' to help provide you with the best experience we can. Cookies are small text files that are placed on your computer or mobile phone when you browse websites.
+
+Our cookies help us:
+
+- Make our website work as you'd expect
+- Remember your settings during and between visits
+- Improve the speed/security of the site
+- Allow you to share pages with social networks like Facebook
+- Continuously improve our website for you
+
+#### c) Sensitive personal data
```

Comment:
Probably need to alter this section too - we'll specifically collect in the case studies form. I have flagged in our cycle tasks that we need them to review this. So probably OK to leave as is here.

Reply:
I've also made a new issue for this - #133 -  but let me know if that's overkill and I can just remove it!